src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -86,7 +86,8 @@ internal class NuGetPackageDownloader : INuGetPackageDownloader
             bool includePreview = false,
             bool includeUnlisted = false,
             DirectoryPath? downloadFolder = null,
-            PackageSourceMapping packageSourceMapping = null)
+            PackageSourceMapping packageSourceMapping = null,
+            bool verifySigning = true)
```

Comment:
What's the difference between the `verifySigning` parameter to this method and the `verifySignatures` constructor parameter?  They sound like the same thing.  I think either we should just have one setting (probably the constructor parameter), or we need very clear comments explaining why we need two different values and what the difference between them is.

Reply:
Thank you very much for the review. Apologize for the duplicated definition of `verifySignatures`. The duplicated is removed and the settings is set in the constructer of `NuGetPackageDownloader`.